homeassistant/components/neurio_energy/sensor.py

```diff
@@ -137,11 +145,22 @@ def __init__(self, data, name, sensor_type, update_call):
         self.update_sensor = update_call
         self._state = None
 
+        self._attr_state_class = STATE_CLASS_MEASUREMENT
```

Comment:
This is static so you can set it as a class attribute on the `NeurioEnergy`  class